runtime/westend/src/lib.rs

```diff
@@ -1086,6 +1086,31 @@ impl pallet_nomination_pools::Config for Runtime {
 	type MaxPointsToBalance = MaxPointsToBalance;
 }
 
+parameter_types! {
+	// The deposit configuration for the singed migration. Specially if you want to allow any signed account to do the migration (see `SignedFilter`, these deposits should be high)
+	pub const MigrationSignedDepositPerItem: Balance = 1 * CENTS;
+	pub const MigrationSignedDepositBase: Balance = 20 * CENTS * 100;
+	pub const MigrationMaxKeyLen: u32 = 512;
+}
+
+impl pallet_state_trie_migration::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type SignedDepositPerItem = MigrationSignedDepositPerItem;
+	type SignedDepositBase = MigrationSignedDepositBase;
+	type ControlOrigin = EnsureRoot<AccountId>;
+	// specific account for the migration, can trigger the signed migrations.
+	type SignedFilter = frame_system::EnsureSignedBy<MigController, AccountId>;
+
+	// Use same weights as substrate ones.
+	type WeightInfo = pallet_state_trie_migration::weights::SubstrateWeight<Runtime>;
+	type MaxKeyLen = MigrationMaxKeyLen;
+}
+
+frame_support::ord_parameter_types! {
+	pub const MigController: AccountId = AccountId::from(hex_literal::hex!("8888888888888888888888888888888888888888888888888888888888888888"));
```

Comment:
When we are running an automatic migration, do we need the controller account?

Reply:
No, but I did not find a good way to disable its declaration (I try to replace it with Sudo but the types are differents).

Reply:
Can you not set signedfilter to ensure root?

Reply:
That's what I tried, but got `expected struct `_sp_runtime::AccountId32`, found `()``.
Just SignedOrigin got a constraint `Success = Self::AccountId`.
Starting trying to map to a dummy account without creating a new controller, but end up thinking that if I would be using a dummy account anyway.
But maybe I can local implement a noop `EnsureOrigin` (thinking twice this looks more harmless than allowing someone even if we know noone have the key).

Reply:
Did this instead https://github.com/paritytech/polkadot/pull/6336/commits/a94fc6a2194590b14a5b75a68c00953179df3711